src/selection.js

```diff
@@ -144,6 +145,9 @@ export default class Selection extends Cursor {
 
   // toggle('<em>')
   toggle (elem) {
+    const elems = content.getTagsByNameAndAttributes(this.host, this.range, elem)
```

Comment:
Maybe these checks could even be moved to `content.js`, wdyt?

Reply:
What function would you add to content?

And I think I don't quite understand the use case here. What do we check here? In what circumstance can you toggle an element in a plainText block?

Reply:
There is a `toggleTag()` function in the content.js file. I thought about moving the guarding code over there because then it would not on a selection. The host element (which is the "block") is passed as an argument. In my understanding it would never make sense to toggle a tag on a block which was enabled with plain text option.

Reply:
>  In my understanding it would never make sense to toggle ON a tag on a block which was enabled with plain text option. But there is no reason why it shouldn't be allowed to toggle OFF a tag in a plain text block.

I would simply disable the option for both cases.

While it may make sense to be able to toggle it off, it would most likely still be weird for a user as you can't undo what you've done. From a UX perspective it's often problematic to allow one-way manipulations. Mostly it's better to simply disable a functionality if it's not fully functional.

Reply:
Also the code is simpler :)